AzFilesHybrid/AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4567,16 +4583,19 @@ function Debug-AzStorageAccountADDSAuth {
 
                 $checks["CheckADObjectPasswordIsCorrect"].Result = "Passed"
                 Write-Verbose "CheckADObjectPasswordIsCorrect - SUCCESS"
+                Write-TestingPassed
             } catch {
                 $checks["CheckADObjectPasswordIsCorrect"].Result = "Failed"
                 $checks["CheckADObjectPasswordIsCorrect"].Issue = $_
-                Write-Error "CheckADObjectPasswordIsCorrect - FAILED"
-                Write-Error $_
+                Write-TestingFailed -Mesage $_ -IsUnexpected $true

```

Comment:
Same thing here, I think it's not necessarily unexpected. This is because `Test-AzStorageAccountADObjectPasswordIsKerbKey` throws an error when it falis. But I don't love that pattern, doesn't let us distinguish test failures from actual exceptional cases that warrant exceptions.